SimDataFormats/CaloAnalysis/src/classes_def.xml

```diff
@@ -0,0 +1,23 @@
+<lcgdict>
+  <class name="CaloParticle"  ClassVersion="4">
+   <version ClassVersion="4" checksum="1423471036"/>
+   <version ClassVersion="3" checksum="4178126599"/>
+   <version ClassVersion="2" checksum="3749156473"/>
```

Comment:
no need for intermediate versions unknown to CMSSW past versions. 
Start from 3.